### PR TITLE
Assets

### PR DESCRIPTION
**Pull request by mhupman**

### CODE REVIEW

File: asset.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	Type            AssetType    `json:"type"`
+	ForLocations    ForLocations `json:"forLocations"`
+	Description     string       `json:"description"`
+	Labels          []string     `json:"labels"`
+	Contents        []Content    `json:"contents"`        // Type:Text
+	PhotoUrl        string       `json:"photoUrl"`        // Type:Photo
+	Details         string       `json:"details"`         // Type:Photo
+	ClickthroughUrl string       `json:"clickthroughUrl"` // Type:Photo
+	AlternateText   string       `json:"alternateText"`   // Type:Photo
+	videoUrl        string       `json:"videoUrl"`        // Type:Video
```
Comment (cdworak) [inline asset.go]: is this one (videoUrl) intentionally lowercase?

**Review by cdworak — APPROVED** [review]

**Comment by mhupman** [comment]
Good catch, fixed!

File: asset_service.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	}
+}
+
+func (a *AssetService) List(opts *ListOptions) (*AssetListResponse, *Response, error) {
+	requrl, err := addListOptions(assetsPath, opts)
+	if err != nil {
+		return nil, nil, err
+	}
+	var v AssetListResponse
+	fmt.Println(requrl)
```
Comment (bhainesva) [inline asset_service.go]: should this be here?

**Review by bhainesva — APPROVED** [review]

File: asset_service.go (hunk 2 above)
Comment (mhupman) [reply asset_service.go]: Derp no, thought I got em all

**Comment by mhupman** [comment]
Hmm I suppose this is very outdated.  Will close.

**Pull request closed without merging**